### ISSUE TITLE
Add support for SMS tag

### ISSUE THREAD

**Issue by lognaturel** [XLSForm/pyxform#207]
pyxform has partial support for SMS sending as defined by https://bitbucket.org/javarosa/javarosa/wiki/SMSSendingAPI. The one piece missing is being able to specify a `tag` for a particular question that would act as the name in the SMS. This will generally be used to specify a shortened name, possibly with a prefix to make it easier to identify as a tag:

```xml
<location tag="+LOC"/>
```

I think this is what `sms_keyword` was meant for but it's only included in the JSON representation and is not converted to XML. 

I propose we make `sms_keyword` output to XML and possibly add an `sms_name` alias to it.

CC @ekigamba @ukanga @jd-alexander

**Comment by jd-alexander** [comment]
Sure this sounds like a good approach. @lognaturel wouldn't mind if you expounded on these with some simple examples so I can be clear on how this should work.

**Comment by lognaturel** [comment]
Good idea.

| type |    name    | sms_name |            label             |
|------|------------|----------|------------------------------|
| text | first_name | +FN      | Please enter your first name |

Would result in the following instance output:
`<first_name tag="+FN" />`

The bind and body would be unaffected. 

Also accepted:
| type |    name    | sms_keyword |            label             |
|------|------------|----------|------------------------------|
| text | first_name | +FN      | Please enter your first name |

**Comment by jd-alexander** [comment]
Okay! So that means I wouldn't need to make any changes to the Javarosa side then to support this since pyxform would be making the conversion to `tag`

**Comment by lognaturel** [comment]
Exactly. Typically the approach has been to make the XLSForm as human-friendly as possible and then apply whatever transformations are needed to match the ODK XForms spec. Which makes me realize that this SMS support is not an official part of that spec. Will file an issue there to discuss whether it should be.
